packages/appstore/src/components/ready-to-update-wallets/ready-to-upgrade-form.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
+import WalletsImage from 'Assets/svgs/wallets';
+import UpgradeInformationList from './upgrade-info-list';
+
+export type TReadyToUpgradeForm = {
+    is_eu: boolean;
+};
+
+const ReadyToUpgradeForm = ({ is_eu }: TReadyToUpgradeForm) => (
+    <React.Fragment>
+        <WalletsImage image={'ReadyToUpdateWalletsIcon'} className='wallet-wrapper--icon' />
+        <div className='wallet-wrapper--text'>
+            <Text size={isMobile() ? 'xsm' : 'm'} align='center' weight='bold' line_height={isMobile() ? 'm' : 'l'}>
```

Comment:
`isMobile() ? 'm' : 'l'` uses in 3 places, better to create a const above

Reply:
Done 😃 👍🏼